src/context.rs

```diff
@@ -71,16 +163,50 @@ impl Context {
     /// ctx.set_val("test", Value::Num(42f32));
```

Comment:
can you update the example here and move this example to `set_val` above? That'd be amazing! :)